app/controllers/twilio_controller.rb

```diff
@@ -0,0 +1,27 @@
+require 'twilio-ruby'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.